localgov_microsites.install

```diff
@@ -56,3 +56,18 @@ function localgov_microsites_update_10001() {
     $role->save();
   }
 }
+
+/**
+ * Update existing sites to enable the theme dependency module for base.
+ */
+function localgov_microsites_update_10002() {
+  // The theme dependency should have caused the module to exist, but lets not
+  // fail if it's not there, as this is just a helper. The theme will continue
+  // to work when updated later, but report the requirement.
+  if (\Drupal::service('theme_handler')->themeExists('localgov_base')) {
+    if (\Drupal::service('module_installer')->install(['localgov_base_helper'])) {
+      return t('Base theme helper module enabled.');
+    }
+    return t('LocalGov Base theme will require the LocalGov Base Helper module. Enable it manually when upgrading to version 1.7.0 or higher. Required by LocalGov Microsites Base 2.0.3');
```

Comment:
Why are we telling people to manually enable the module here, rather than enabling it for them?

```
if (!\Drupal::service('module_handler')->moduleExists('localgov_base_helper')) {
    \Drupal::service('module_installer')->install(['localgov_base_helper']);
  }
```

Why don't we do that ^ - check if the module exists, and if so enable it?